### PR TITLE
1.2 plat 5838 live consistency tests

### PR DESCRIPTION
**Pull request by lilachmaliniak**

### CODE REVIEW

File: common/PersistenceFormat.js [hunk 1]
```diff
@@ -61,52 +67,151 @@ class PersistenceFormatBase {
...
-            var matched = tsChunktMatch.exec(tsChunkName);
-            if (matched) {
-                return matched[1] + '-' + matched[2] + '.mp4';
+if (!preserveOriginalHLS.enable) {
+    module.exports = new PersistenceFormatBase();
+} else {
+
+    function getEntryPathHelper(entryPath) {
+
+        let basePath = `${rootFolder}/${entryPath}-`;
```
Comment (igorshevach) [inline common/PersistenceFormat.js]: please use path.join whenever you build paths

File: common/PersistenceFormat.js [hunk 2]
```diff
@@ -61,52 +67,151 @@ class PersistenceFormatBase {
...
+
+
+}
+
 
-        compressChunkName(tsChunkName) {
-            var matched = tsChunktMatch.exec(tsChunkName);
-            if (matched) {
-                return matched[1] + '-' + matched[2] + '.mp4';
+if (!preserveOriginalHLS.enable) {
```
Comment (igorshevach) [inline common/PersistenceFormat.js]: it looks like checking for dirHierarchy && preserveOriginalHLS.enable will simplify the code

File: common/PersistenceFormat.js (hunk 1 above)
Comment (lilachmaliniak) [inline common/PersistenceFormat.js]: done in all relevant places.

File: common/PersistenceFormat.js (hunk 2 above)
Comment (lilachmaliniak) [inline common/PersistenceFormat.js]: changed logic the base class will be created either if hls preserve is disabled or if using hierarchy of files. Renamed the class used when hls is perserved without hierarchy from preserveOriginalHLSFormat to SimplfiedPersistenceFormat.

**Comment by lilachmaliniak** [comment]
will be merged to 1.3.

**Pull request closed without merging**